2022-round-1/29riyasaxena/nx_version.txt

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
Thanks @29riyasaxena!
Could you please update this to the latest released version of NetworkX, 2.7.1
Please do also check your python version

Reply:
Okay, Let me check it!

Reply:
Hi! I modified the file.